readthedocs/privacy/backends/syncers.py

```diff
@@ -8,12 +14,17 @@
 log = logging.getLogger(__name__)
 
 
+# The file builting was removed in Python 3, so we should not use it
+# Setting to None here to silence pylint for this builtin only
+file = None  # pylint: disable=redefined-builtin
```

Comment:
Perhaps instead this should just rename the `file` argument on the various `copy` methods to not use a builtin. A quick search shows this as the only mention of the `file` argument:

https://github.com/rtfd/readthedocs.org/blob/b9f15d9172df2d2d0c2f5fc132ba9c1b25faf709/readthedocs/projects/tasks.py#L857